source/getting-started/installation-raspberry-pi-image.markdown

```diff
@@ -26,13 +26,25 @@ Home Assistant will now be available by navigating with a browser to `http://ip-
 The following extras are included on the image:
 
  - GPIO pins are ready to use.
- - Mosquitto MQTT broker is installed (not activated by default).
  - Bluetooth is ready to use (supported models only, no Bluetooth LE).
+ - Hassbian-scrips. A set of script written to add extra functionality to your raspberry pi installation.  
```

Comment:
Typo in `Hassbian-scrips`

You also usually use `Raspberry Pi` and not `raspberry pi`